app/i18n/en/index.ts

```diff
@@ -2036,7 +2036,11 @@ const en: BaseTranslation = {
     registerTitle: "Need to upgrade your account",
     registerContent: "Register with your phone number to receive sats",
     notYet: "Not yet",
-    availableTomorrow: "This section will be available tomorrow!"
+    availableTomorrow: "This section will be available tomorrow!",
+    motivatingBadger1: 'Dig your way',
```

Comment:
is there a way to do that: `motivatingBadger: Dig your way\n\nthrough these\n\n...` ?
that would make it easier for the translators to understand the intent. not a big deal if that's not practical

Reply:
Kinda sorta. I was thinking we could translate it as 1 entity, and then i'd divide it into 3 or 4 lines with a maximum amount of characters per line. I'll do that.

Reply:
Nvm. Trying to do calculate all that was ridiculous. I came up with another solution. See below